e3nn/util/dataset/molecules.py

```diff
@@ -149,3 +149,135 @@ def download(self):
 
         self.files = sorted(glob.glob(os.path.join(xyz_path, "*.xyz")))
         assert len(self.files) == 133885
+
```

Comment:
I think these changes should be removed from this pull request since they don't have to do with the cuda implementation of the message_passing convolution.